mobile_back_pixel/cpp/backend_tflite/resize_argmax_op.cc

```diff
@@ -0,0 +1,348 @@
+#include <string.h>
```

Comment:
File needs a copyright heaer

Reply:
Do you want the backend to be copyright MLPerf Authors or copyright Google?

Reply:
Done. Added Google Apache Header